frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -25,10 +37,11 @@ const MultiColumnFieldRow: React.FC<MultiColumnFieldRowProps> = ({
         </div>
       );
     })}
-    <div className="odc-multi-column-field__col--button">
-      <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
-      <span className="sr-only">Delete</span>
-    </div>
+    {toolTip ? (
+      <Tooltip content={toolTip}>{renderMinusCircleIcon(onDelete)}</Tooltip>
```

Comment:
Why are we displaying a tooltip? No where else ini console do we do this right now. 

cc @serenamarie125